sdk/inc/azure/core/az_mqtt5_rpc.h

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ *
+ * @brief Common for az_mqtt5_rpc server and client.
+ *
+ * @note You MUST NOT use any symbols (macros, functions, structures, enums, etc.)
+ * prefixed with an underscore ('_') directly in your application code. These symbols
+ * are part of Azure SDK's internal implementation; we do not document these symbols
+ * and they are subject to change in future versions of the SDK which would break your code.
+ */
+
+#ifndef _az_MQTT5_RPC_H
+#define _az_MQTT5_RPC_H
+
+#include <azure/core/az_mqtt5_connection.h>
+#include <azure/core/az_result.h>
+#include <azure/core/az_span.h>
+
+#include <azure/core/_az_cfg_prefix.h>
+
+/**
+ * @brief The default timeout in seconds for subscribing.
```

Comment:
I haven't added the timeout yet for publishing, but that's a good idea to share the timer functionality for both since they can't happen at the same time!

Reply:
updated & added the timer